client/main.go

```diff
@@ -140,6 +88,15 @@ func TinkHardwareClient() (hardware.HardwareServiceClient, error) {
 	return hardware.NewHardwareServiceClient(conn), nil
 }
 
+// TinkTemplateClient creates a new hardware client.
+func TinkTemplateClient() (template.TemplateServiceClient, error) {
+	conn, err := GetConnection()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Packet logger is used everywhere else, why use stdlib logger?

Also, why do we not return the error here and the other client methods?

Reply:
done